odbc_fdw.c

```diff
@@ -986,60 +971,6 @@ static ForeignScan* odbcGetForeignPlan(PlannerInfo *root, RelOptInfo *baserel,
                                 NULL /* outer_plan */ );
 }
 
-/* routines for versions older than 9.2.0 */
-#else
-
-/*
- * odbcPlanForeignScan
- *		Create a FdwPlan for a scan on the foreign table
- */
-static FdwPlan *
-odbcPlanForeignScan(Oid foreigntableid, PlannerInfo *root, RelOptInfo *baserel)
-{
-    FdwPlan	*fdwplan;
-    unsigned int table_size	= 0;
-    char *svr_dsn			  = NULL;
-    char *svr_driver	  = NULL;
-    char *svr_host    	= NULL;
-    char *svr_port    	= NULL;
-    char *svr_database	= NULL;
-    char *svr_schema		= NULL;
-    char *svr_table			= NULL;
-    char *sql_query			= NULL;
-    char *sql_count			= NULL;
-    char *username			= NULL;
-    char *password			= NULL;
-    List *col_mapping_list;
-
-#ifdef DEBUG
-    elog(NOTICE, "odbcPlanForeignScan");
-#endif
-
-    /* Fetch the foreign table options */
-    odbcGetOptions(foreigntableid, &svr_dsn, &svr_driver, &svr_host, &svr_port,
-                   &svr_database, &svr_schema, &svr_table, &sql_query,
-                   &sql_count, &username, &password, &col_mapping_list);
-
-    fdwplan = makeNode(FdwPlan);
-    fdwplan->startup_cost = 10;
-    fdwplan->total_cost = 100 + fdwplan->startup_cost;
-    fdwplan->fdw_private = NIL;	/* not used */
-
-#ifdef DEBUG
-    elog(NOTICE, "new total cost: %f", fdwplan->total_cost);
-#endif
-
-    odbcGetTableSize(svr_dsn, svr_driver, svr_host, svr_port, svr_database, svr_schema, svr_table, username, password, sql_count, &table_size);
```

Comment:
Yes, I'm assuming it's not valid for PG>=9.02 so dismissing it. It'll be nice to rewrite it for PG 9.5